packages/discord.js/src/util/Options.js

```diff
@@ -153,31 +168,16 @@ class Options extends null {
    * * `GuildChannelManager` - Sweep archived threads
    * * `ThreadManager` - Sweep archived threads
    * <info>If you want to keep default behavior and add on top of it you can use this object and add on to it, e.g.
-   * `makeCache: Options.cacheWithLimits({ ...Options.defaultMakeCacheSettings, ReactionManager: 0 })`</info>
+   * `makeCache: Options.cacheWithLimits({ ...Options.DefaultMakeCacheSettings, ReactionManager: 0 })`</info>
    * @type {Object<string, LimitedCollectionOptions|number>}
    */
-  static get defaultMakeCacheSettings() {
+  static get DefaultMakeCacheSettings() {
```

Comment:
If the casing on these is changed then the `BitField`s `defaultBit` should be changed too